File: P2A2 Algoritmos a Lenguajes/costo_cuaderno.cpp
//Nombre del alumno: Enrique Arellano Valmaña
//ID: 00624255
//Carrera: Ing Animación Digital

#include <iostream>

using namespace std;

int main (){
    
    double Num_Hojas;
    
    cout << "¿Número de hojas del cuaderno?: ";
    cin >> Num_Hojas;
    
    if (Num_Hojas == 250){
        cout << "El cuaderno cuesta $16.00";}
    else{
        if (Num_Hojas == 100){
            cout << "El cuaderno cuesta $11.00";}
        else{
            if (Num_Hojas == 80){
               cout << "El cuaderno cuesta $8.00";}
            else{
                if (Num_Hojas == 50){
                   cout << "El cuaderno cuesta $4.50";}
                else {
                    cout << "Número de hojas incorrecto.";
                }
            }
        }
    }
    
    
    
    
    
    
    return 0;
}
    

File: P2A2 Algoritmos a Lenguajes/costo_cuaderno.py
#Enrique Arellano Valmaña
#00624255
#Ing Animacion Digital

print ("¿Número de hojas del cuaderno?: ")
Num_Hojas = int(input())

if Num_Hojas == 250:
    print("El cuaderno cuesta $16.00.")
elif Num_Hojas == 100:
    print("El cuaderno cuesta $11.00.")
elif Num_Hojas == 80:
    print("El cuaderno cuesta $8.00.")
elif Num_Hojas == 50:
    print("El cuaderno cuesta $4.50.")
else:
    print("Número de hojas incorrecto")